frontend/packages/operator-lifecycle-manager/src/components/modals/uninstall-operator-modal.tsx

```diff
@@ -88,72 +152,330 @@ export const UninstallOperatorModal: React.FC<UninstallOperatorModalProps> = ({
+      isInline
+    >
+      <p>
+        <Trans t={t} ns="olm">
+          Operator <strong>{{ name }}</strong> successfully uninstalled from{' '}
+          <strong>{{ namespace }}</strong>.
+        </Trans>
+      </p>
+    </Alert>
+  );
+};
+
+const OperatorUninstallErrorAlert: React.FC<{ errorMessage: string }> = ({ errorMessage }) => {
+  const { t } = useTranslation();
+  return (
+    <Alert
+      variant="danger"
+      className="co-alert"
+      title={t('olm~Error uninstalling Operator')}
+      isInline
+    >
+      <p>
+        {t('olm~There was an error uninstalling the operator.')}
+        <br />
+        {errorMessage}
+      </p>
+    </Alert>
+  );
+};
+
+const OperandDeletionErrorAlert: React.FC<{
+  operandDeletionErrors: OperandWithErrorProps[];
+  csvName: string;
+  cancel?: () => void;
+}> = ({ operandDeletionErrors, csvName, cancel }) => {
+  const { t } = useTranslation();
+  return (
+    <Alert variant="danger" className="co-alert" title={t('olm~Error deleting Operands')} isInline>
+      <p>
+        {t(
+          'olm~There were errors deleting the following Operands, they will need to be deleted manually:',
+        )}
+      </p>
+      <OperandErrors
+        operandsWithErrors={operandDeletionErrors}
+        csvName={csvName}
+        cancel={cancel} // for breadcrumbs & cancel modal when clicking on operand links
+      />
+    </Alert>
+  );
+};
+
+const OperandDeletionSuccessAlert: React.FC<{ name: string; namespace: string }> = ({
+  name,
+  namespace,
+}) => {
+  const { t } = useTranslation();
+  return (
+    <Alert
+      variant="success"
+      className="co-alert"
+      title={t('olm~Successfully deleted all Operand Instances')}
+      isInline
+    >
+      <p>
+        <Trans t={t} ns="olm">
+          All Operand instances for Operator <strong>{{ name }}</strong> in{' '}
+          <strong>{{ namespace }}</strong> have been deleted.
+        </Trans>
+      </p>
+    </Alert>
+  );
+};
+
+const UninstallAlert: React.FC<{ errorMessage: string; name: string; namespace: string }> = ({
+  errorMessage,
+  name,
+  namespace,
+}) =>
+  errorMessage ? (
+    <OperatorUninstallErrorAlert errorMessage={errorMessage} />
+  ) : (
+    <OperatorUninstallSuccessAlert name={name} namespace={namespace} />
+  );
+
+const OperandsTable: React.FC<OperandsTableProps> = ({ operands, loaded, csvName, cancel }) => {
+  const { t } = useTranslation();
+  return (
+    <StatusBox
+      skeleton={<div className="loading-skeleton--table" />}
+      data={operands}
+      loaded={loaded}
+    >
+      <table className="pf-c-table pf-m-compact pf-m-border-rows">
+        <thead>
+          <tr>
+            <th>{t('olm~Name')}</th>
+            <th>{t('olm~Kind')}</th>
+            <th>{t('olm~Namespace')}</th>
+          </tr>
+        </thead>
+        <tbody>
+          {operands
+            .sort((a, b) => a.metadata.name.localeCompare(b.metadata.name))
```

Comment:
We want a stable sort here, so we should consider secondary sorts of namespace and then kind.

Reply:
Hi @spadgett, do you mean sort by name, and if there're equal, then sort by namespace, then kind?  Similar to this [podCompare](https://github.com/openshift/console/blob/a3d3f4de592a736e9e6a8f952a757285670bfb3c/frontend/public/components/overview/pods-overview.tsx#L78).

Reply:
Hi @spadgett, addressed latest comments, except this last one ^^  
thanks :smile: